pkg/monitoring/metrics/common/workqueue/metrics.go

```diff
@@ -69,7 +76,7 @@ func (_ prometheusMetricsProvider) NewLatencyMetric(name string) k8sworkqueue.Hi
 	return latency
 }
 
-func (_ prometheusMetricsProvider) NewWorkDurationMetric(name string) k8sworkqueue.HistogramMetric {
+func (Provider) NewWorkDurationMetric(name string) k8sworkqueue.HistogramMetric {
```

Comment:
This may be out of scope, but I wonder what if these operations take much longer, 30 seconds? who knows how it might behave under load... @xpivarc is it a concern?

Reply:
@vladikr What do you mean?